releases.md

```diff
@@ -0,0 +1,201 @@
+# Talos II support in coreboot
+
+## Key changes
+
+1. Added RAM initialization
+2. Added SCOM registers support
+
+---
+## Statistics
+
+Since bootblock release: \
+<span style="color:yellow">59</span> files were changed including <span style="color:lightgreen">48</span> new files created \
+<span style="color:lightgreen">12407</span> lines of code were added \
+<span style="color:orangered">76</span> lines of code were removed
+
+Check the statistics with:
+```
+git diff --stat 6f5a735df6 c0fe1bbd4d
```

Comment:
This (and probably statistics above) will change after last fix is merged. Leaving this comment as a reminder, please don't mark it resolved until it's done.

Reply:
I have looked for some tool to gather fun statistics. It's hard to find something working for a specific branch, but [gutstats](https://www.systutorials.com/docs/linux/man/1-gitstats/) looks promising.

Reply:
@IgorBagnucki https://github.com/3mdeb/coreboot/pull/57 is merged. Do we need to update something here?

Reply:
I have updated the statistics after PR:57. I have also removed  the information about new files created, because I couldn't verify that the value is correct.

Reply:
Updated after PR:58 too